src/main/java/com/projetonetflix/projetonetflix/dto/DirectorTitleDTO.java

```diff
@@ -0,0 +1,56 @@
+package com.projetonetflix.projetonetflix.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.projetonetflix.projetonetflix.model.entity.Director;
+import com.projetonetflix.projetonetflix.model.entity.Title;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DirectorTitleDTO {
+
+    private String name;
+    private int age;
+
+    @JsonIgnore
+    private List<TitleDTO> titleList;
```

Comment:
olha, eu entendi assim: essa classe é o diretor dentro de um tituloDto.
Se for esse caso, como ela pertence ao filme, não teria porque ela ter info de filmes.

Reply:
ai como essa classe é um classe de resposta, não teria porque ter o json ignore e nem o campo, já que o json ignore significa que a biblioteca jackson não deve colocar esse campo no json de resposta. 
Mas essa é um classe de resposta. ela só serve para isso. Não precisamos colocar um campo nele e depois não exibir.

Reply:
Ah, curiosidade, o DTO é o padrão (pattern), mas nem todo DTO precisa ser nomeado DTO.
Aqui no montserrat muitas vezes a gente dá um nome mais semântico.
Um DTO de resposta de controller ganha o sufixo de Response, ao invés de DTO

exemplo: TitleDto vira TitleResponse. 
ai só de ler o nome já sabemos que essa é uma classe de resposta de controller.
tipo, se você tiver um endpoint post ele recebe um payload. Ai criamos um dto de entrada, request
ai teriamos por exemplo um tituloRequest, e só de ler o nome da classe sabemos que é um classe parecer parametros

Reply:
Coloquei assim para ficar mais fácil o meu entendimento, mas bom saber como é na pags, posso fazer essa alteração também.